adapters/tappx/tappxtest/exemplary/single-banner-impression-extra.json

```diff
@@ -0,0 +1,124 @@
+{
+  "mockBidRequest": {
+    "id": "0000000000001",
+    "test": 1,
+    "imp": [
+      {
+        "id": "adunit-1",
+        "banner": {
+          "w": 300,
+          "h": 250
+        },
+        "ext": {
+         "bidder": {
+            "tappxkey": "pub-12345-android-9876",
+            "endpoint": "ZZ123456PS",
+            "host": "testing.ssp.tappx.com/rtb/v2/",
+            "mktag": "mktag-12345-android-9876"
+          }
+        }
+      }
+    ],
+    "app": {
+      "id": "app_001",
+      "bundle": "com.rovio.angrybirds",
+      "publisher": {
+        "id": "2"
+      }
+    },
+    "user": {
+      "buyeruid": "A-38327932832"
+    }
+  },
+
+  "httpCalls": [
+    {
+      "expectedRequest": {
+        "uri": "http://testing.ssp.tappx.com/rtb/v2/ZZ123456PS?tappxkey=pub-12345-android-9876&type_cnn=prebid&v=1.3",
+        "body": {
+          "id": "0000000000001",
+          "test": 1,
+          "imp": [
+            {
+              "id": "adunit-1",
+              "banner": {
+                "w": 300,
+                "h": 250
+              },
+              "ext": {
+                "bidder": {
+                  "tappxkey": "pub-12345-android-9876",
+                  "endpoint": "ZZ123456PS",
+                  "host": "testing.ssp.tappx.com/rtb/v2/",
+                  "mktag": "mktag-12345-android-9876"
+                }
+              }
+            }
+          ],
+          "app": {
+            "bundle": "com.rovio.angrybirds",
+            "id": "app_001",
+            "publisher": {
+              "id": "2"
+            }
+          },
+          "user": {
+            "buyeruid": "A-38327932832"
+          },
+          "ext": {
+            "bidder": {
+              "tappxkey": "pub-12345-android-9876",
+              "mktag": "mktag-12345-android-9876"
+            }
+          }
+        }
+      },
+      "mockResponse": {
+        "status": 200,
+        "body": {
+          "id": "75472df2-1cb3-4f8e-9a28-10cb95fe05a4",
+          "seatbid": [{
+              "bid": [{
+                  "id": "wehM-93KGr0_0_0",
+                  "impid": "adunit-1",
+                  "price": 0.5,
+                  "cid": "3706",
+                  "crid": "19005",
+                  "adid": "19005",
+                  "adm": "<!-- admarkup -->",
+                  "cat": ["IAB2"],
+                  "adomain": ["test.com"],
+                  "h": 250,
+                  "w": 300
+              }]
+          }],
+          "bidid": "wehM-93KGr0"
+        }
+      }
+    }
+  ],
+
+  "expectedBidResponses": [
+    {
+      "currency": "USD",
+      "bids": [
+        {
+          "bid": {
+            "id": "wehM-93KGr0_0_0",
+            "impid": "adunit-1",
+            "price": 0.5,
+            "adm": "<!-- admarkup -->",
```

Comment:
Thank you for using a fake ad markup value for tests.

Reply:
Thank you for all the help and detail attention! 🤗